nan_callbacks.h

```diff
@@ -101,7 +101,7 @@ static const int kIndexPropertyQueryIndex =      5;
 static const int kIndexPropertyFieldCount =      6;
 
 template<typename T, typename P>
-P *GetWrapper(T needle) {
+static inline P *GetWrapper(T needle) {
```

Comment:
For reference, this is the stacktrace for the SIGSEGV when the static is removed.

```
PID 94064 received SIGSEGV for address: 0x0
0   segfault-handler.node               0x0000000100ffa90f _ZL16segfault_handleriP9__siginfoPv + 287
1   libsystem_platform.dylib            0x00007fff8e80a5aa _sigtramp + 26
2   ???                                 0x0000000000000783 0x0 + 1923
3   binding.node                        0x0000000103107d8c _ZNSt3__16__treeINS_12__value_typeIPFvRKN3Nan20FunctionCallbackInfoIN2v85ValueEEEEPNS2_3imp15FunctionWrapperEEENS_19__map_value_compareISA_SE_NS_4lessISA_EELb1EEENS_9allocatorISE_EEE20__node_insert_uniqueENS_21__tree_const_iteratorISE_PNS_11__tree_nodeISE_PvEElEESQ_ + 56
4   binding.node                        0x0000000103107d32 _ZNSt3__16__treeINS_12__value_typeIPFvRKN3Nan20FunctionCallbackInfoIN2v85ValueEEEEPNS2_3imp15FunctionWrapperEEENS_19__map_value_compareISA_SE_NS_4lessISA_EELb1EEENS_9allocatorISE_EEE15__insert_uniqueINS_4pairISA_SD_EEEENS_15__tree_iteratorISE_PNS_11__tree_nodeISE_PvEElEENS_21__tree_const_iteratorISE_ST_lEEOT_ + 62
5   binding.node                        0x0000000103107c3b _ZN3Nan3imp7FactoryIN2v816FunctionTemplateEE3NewEPFvRKNS_20FunctionCallbackInfoINS2_5ValueEEEENS2_5LocalIS6_EENSC_INS2_9SignatureEEE + 235
6   binding.node                        0x00000001031023c1 _Z14RegisterModuleN2v86HandleINS_6ObjectEEE + 52
7   node                                0x0000000100601047 _ZN4node6DLOpenERKN2v820FunctionCallbackInfoINS0_5ValueEEE + 725
8   node                                0x00000001001749ee _ZN2v88internal25FunctionCallbackArguments4CallEPFvRKNS_20FunctionCallbackInfoINS_5ValueEEEE + 158
9   node                                0x000000010019c97b _ZN2v88internalL19HandleApiCallHelperILb0EEENS0_11MaybeHandleINS0_6ObjectEEEPNS0_7IsolateERNS0_12_GLOBAL__N_116BuiltinArgumentsILNS0_21BuiltinExtraArgumentsE1EEE + 811
10  node                                0x000000010019f1cd _ZN2v88internalL21Builtin_HandleApiCallEiPPNS0_6ObjectEPNS0_7IsolateE + 61
11  ???                                 0x000039d1e69060bb 0x0 + 63573679169723
```

_It's bedtime here, I'll check back in a couple hours_